ops/terraform/services/eft/lambda_src/sftp_outbound_transfer/dev-requirements.txt

```diff
@@ -1,87 +1,89 @@
 #
-# This file is autogenerated by pip-compile with Python 3.12
+# This file is autogenerated by pip-compile with Python 3.9
```

Comment:
This takes us from 3.12 to 3.9? Is this intentional?

Reply:
Just complied with Python 3.9 instead of 3.12. If you want, I can upgrade my Python version to 3.12 and push that up. I am not sure if there would be any difference here other than this commented-out line.

Reply:
Complied with 3.12 with latest commit